Brandon_hopper.asm

```diff
@@ -0,0 +1,228 @@
+
+	      jr   $ra              # return from subroutine
+	# End of subroutine to print the numbers on one line
+	###############################################################
+
+
+##########################################################
+# .data segment for Problem 2
+##########################################################
+.data
+
+Print_Info: .asciiz "\n\n####################\nStarting Problem 2\n##################\nName: Brandon Hopper\nTitle: Printing Program\nDescription: This program was designed to print stuff xD." 
+Ask_input1: .asciiz "\nPlease enter a number for a: "
+Ask_input2: .asciiz "\nPlease enter a number for b: "
+Inser_data: .word 4 #Load word into memory
+
+
+#---------------------------------------------------------
+# Problem # 2
+#---------------------------------------------------------
+.text
+	#Print the info to the screen
+problem2: la $a0, Print_Info
+	li $v0, 4
+	syscall
+	
+	#Print message to ask for user input 1
+	li $v0, 4
+	la $a0, Ask_input1
+	syscall
+		
+	#Prompt for user input 1
+	li $v0,5
+	syscall
+        move $t0, $v0
+        
+	#Print message to ask for user input 2
+	li $v0, 4
+	la $a0, Ask_input2
+	syscall
+		
+	#Prompt for user input 2
+	li $v0,5
+	syscall
+        move $t1, $v0
+        
+        li $v0, 1
+        li $t1, 4
+	syscall
+        
+        add $t3,$zero,$t0 #set t3 to t1s value
+        add $t4,$zero,$t1 #set t4 to t0s value
+        
+        #Multiply and save the results
+mult:   add $t3,$t3,$t3
+	addi $t4, $t4, -1     # decrement loop counter
+	bgtz $t4, mult        # repeat while not finished
+        
+        #Add and set
+	add $t5,$t1,$t0
+	
+	#Subtract and set
+	sub $t6,$t1,$t0
+                 
+        move $a0, $t4
+        #add $a0,$t1,$t0
+        #print the values
+        li $v0, 1
+        li $a0, 4
+	syscall
+	
+	#print the values
+       # li $v0, 1
+       # li $t5, 5
+	#syscall
+	
+	#print the values
+       # li $v0, 1
+       # li $t1, 5
+	#syscall
+	
+	j problem3
+	      
+	li   $v0, 10          # system call for exit
+	syscall               
+###############################################################
+
+
+##########################################################
+# .data segment for Problem 2
+##########################################################
+.data
+Print_Info2: .asciiz "\n\n####################\nStarting Problem 3\n##################\n"
+
+#---------------------------------------------------------
+# Problem # 3
+#---------------------------------------------------------
+.text
+	#Print info on screen for which problem your on
+problem3:la $a0, Print_Info2
+	li $v0, 4
+	syscall
+	
+	addi $t0, $zero, 8
+        addi $t1, $zero, 1
+        addi $t2, $zero, 3
+        #Add odd numbers
+addodds:add $t1,$t1,$t2
+	addi $t2, $t2, 2      # keep adding 2
+	addi $t0, $t0, -1     # decrement loop counter
+	bgtz $t0, addodds        # repeat while not finished
+
+        #print the values
+        li $v0, 1
+        lw   $a0, 0($t1) 
```

Comment:
-10 Your program is crashing because of line 27
	lw $a0, 0($t1)
is saying "Load the word at address of ($t1) with 0 offset", but $t1 is NOT an address, it holds a literal value.  If you look at the error message it says Runtime exception at 0x00400030: fetch address not aligned on word boundary 0x00000051